content/en/get-started/store-and-retrieve/set-up/index.md

```diff
@@ -273,16 +269,25 @@ DataCap acts as a kind of _multiplier_ for block rewards. If a storage provider
 Signing up to Filecoin Plus is easy and free!
 
 {{< alert icon="tip" >}}
-You need a GitHub account that is at least 180 days old. If you don't have a GitHub account that's old enough, [get in touch with the team on Filecoin Slack](https://filecoin.io/slack/).
+You need a GitHub account that is at least 180 days old. If you don't have a GitHub account that's old enough, [get in touch with #fil-lotus-help on Filecoin Slack](https://filecoinproject.slack.com/archives/CPFTWMY7N).
```

Comment:
The issue with linking directly to a particular Slack message, is that messages get deleted after a few months. I recommend just using the regular Slack link.